sputnikdao-factory2/src/lib.rs

```diff
@@ -432,4 +433,73 @@ mod tests {
             vec![format!("test.{}", accounts(0)).parse().unwrap()]
         );
     }
+
+    //              #################################              //
+    //              #    Factory ownership tests    #              //
+    //              #################################              //
+
+    #[test]
+    fn test_factory_can_get_current_owner() {
+        let mut context = VMContextBuilder::new();
+        testing_env!(context
+            .current_account_id(alice())
+            .predecessor_account_id(alice())
+            .attached_deposit(10)
+            .build());
+        let mut factory = SputnikDAOFactory::new();
+
+        factory.create(alice(), "{}".as_bytes().to_vec().into());
```

Comment:
@nninkovicSQA  - this is a method to create a DAO. 
For testing the factory, `SputnikDAOFactory::new();` is the method used for a new factory, which will define the owner as the same account this factory code is deployed to until changed via `set_owner`.